msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfigurationFactory.java

```diff
@@ -50,6 +52,7 @@
 
 public class PublicClientApplicationConfigurationFactory {
     private static final String TAG = PublicClientApplicationConfigurationFactory.class.getSimpleName();
+    private static final int DEFAULT_MAX_THREAD_POOL_INTERACTIVE_CONCURRENT = 5;
```

Comment:
Just curious is there any specific reason for choosing 5 here?

Reply:
Nope.  Open to suggestion, but didn't seem reasonable for it to be unlimited.  We used 5 for the silent pool in command dispatcher, so i made it match.